python/lsst/obs/base/fitsRawFormatterBase.py

```diff
@@ -163,6 +163,10 @@ def makeWcs(self, visitInfo, detector):
             Raised if there is an error generating the SkyWcs, chained from the
             lower-level exception if available.
         """
+        if self.observationInfo.tracking_radec is None:
+            # This is not an on-sky observation
+            return None
+
```

Comment:
Might be a good idea to put this behind a function or `ObservationInfo` method, so if we need to expand it to e.g. `tracking_radec is None or <observation type is SCIENCE>` in order to make it more robust, we only have to do it in one place.